src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -358,7 +358,27 @@ functionsToRunOnDataTablesDrawEvent: [],
          crud.functionsToRunOnDataTablesDrawEvent.forEach(function(functionName) {
             crud.executeFunctionByName(functionName);
          });
-      } ).dataTable();
+          @if($crud->getOperationSetting('actionsColumnAsDropdown'))
+          // Get last column
+          $('#crudTable').find('td:last-child').map((index, el) => {
+              const actionButtons = $(el).find('a.btn.btn-link');
+              if (actionButtons.length === 0) {
```

Comment:
We rely on finding buttons to check if it's the "actions" column. 

Can't we add some property to that column when we add it to the column stack in https://github.com/Laravel-Backpack/CRUD/blob/32c5d60310c13098fd6d6e012711d11b673decce/src/resources/views/crud/list.blade.php#L132 ?

That way we are guaranteed to get the actions column if it's visible right ?